src/main/java/org/folio/inventory/common/WebContext.java

```diff
@@ -25,6 +25,11 @@ public String getOkapiLocation() {
     return getHeader("X-Okapi-Url", "");
   }
 
+  @Override
+  public String getUserId() {
+    return getHeader("X-Okapi-User-Id", "");
```

Comment:
Maybe it will be a good idea to move all strings with headers names to the constants.

Reply:
Moved to the constants.